vortex-file/src/read/mod.rs

```diff
@@ -59,28 +59,19 @@ pub type MessageId = Vec<LayoutPartId>;
 pub struct MessageLocator(pub MessageId, pub ByteRange);
 
 #[derive(Debug)]
-pub enum BatchRead {
+pub enum PollRead<T> {
```

Comment:
How about PollRead has `None` option as well? So you don't have `PollRead<Option<T>>` but maybe it' more obvious